index.md

```diff
@@ -1,25 +1,49 @@
 ---
 layout: front
-title: META MESH | Building the next Internet
+title: META MESH | Wireless Networking for All
 ---
-# META MESH is building a wireless mesh network in Pittsburgh, Pennsylvania.
-
-**What is a mesh network?** Most of us have a home Internet connection to an
-Internet Service Provider (ISP) such as Verizon or Comcast. To communicate with
-each other, all our network traffic must go through these ISPs &mdash; even if
-the person to whom we are sending an e-mail lives across the street. This
-centralized way of networking presents many security and privacy problems.
-
-A *mesh network* cuts out that centralized middleman. Instead of traffic having
-to go to a big server hundreds of miles away, a mesh network's traffic is routed
-by the shortest and fastest possible path.
-
-META MESH is building a network of
-wireless routers to make this model of networking a reality in the Pittsburgh
-area. That network is aptly named [**PittMesh**](http://pittmesh.net). We don't
-use expensive equipment to make this happen- in fact we've designed our network
-so that anyone can join it with consumer-grade, off-the-shelf Wi-Fi hardware
-available at any electronics store. When you join the PittMesh Network, you not
+# META MESH empowers people to build Community Wireless Networks
+
+**What is a Community Wireless Network?** 
+
+People who have a home Internet connection lease their ability to communicate
+with others to centralized ISP's like Comcast, Verizon, and Time Warner. Other people do not have Internet access at all because for them it is a luxury or ISPs do not service their neighborhoods. Two sides of a coin, two problems.
+
+What if we could solve both at the same time?What if residents came together and built their own infrastructure, 
+funded not by a corporation or even the local government but by the users themselves? Meta Mesh thinks they can by using Community Wireless Networks.
+
+Much like an office's network, resources can be available even without Internet 
+access. You don't need Internet access to use your network printer, right? So why should you need Internet access to send a file to your friend three blocks away? You can both put an inexpensive wireless router and antenna on your roofs, walls, chimneys, or even in your windows, configure them the right way, and you without paying an ISP a dime you have your own wireless network.
+
+And what if, by building that wireless network you opened it up, so that anyone could use it for Internet access? After all, you don't get a refund for bandwidth you don't use, so you might as well give it away-- just like your local coffeeshop does. And by setting up these anteannas and giving away that bandwidth, other people benefit who might not have Internet connections at home. 
```

Comment:
You could try &mdash;` in place of the `--` or just put a real long dash in there via charmap.

Or copy paste this: `–`.